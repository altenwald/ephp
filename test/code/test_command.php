<?php

$filter = 'ephp';
var_dump(trim(`cat src/ephp.app.src | grep $filter | head -1`));
$result = trim(`echo \``);
var_dump(strpos($result, "sh:") !== FALSE);
var_dump(strpos($result, "yntax error") !== FALSE);
