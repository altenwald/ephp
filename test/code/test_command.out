string(21) "{application, ephp, ["
string(50) "sh: 1: Syntax error: EOF in backquote substitution"
